sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -2,32 +2,117 @@
 // Licensed under the MIT License.
 
 import * as coreHttp from "@azure/core-http";
-import {
-  JsonWebKey,
-  JsonWebKeyOperation,
-  JsonWebKeyCurveName,
-  JsonWebKeyType
-} from "./core/models";
+import { JsonWebKeyOperation, JsonWebKeyCurveName, JsonWebKeyType } from "./core/models";
 import { DeletionRecoveryLevel } from "./core/models";
 
 /**
  * @interface
  * An interface representing the key client. For internal use.
  */
 export interface KeyClientInterface {
+  /**
+   * Recovers the deleted key in the specified vault. This operation can only be performed on a
+   * soft-delete enabled vault.
+   */
   recoverDeletedKey(name: string, options?: GetDeletedKeyOptions): Promise<KeyVaultKey>;
+  /**
+   * The get method gets a specified key and is applicable to any key stored in Azure Key Vault.
+   * This operation requires the keys/get permission.
+   */
   getKey(name: string, options?: GetKeyOptions): Promise<KeyVaultKey>;
+  /**
+   * The delete operation applies to any key stored in Azure Key Vault. Individual versions
+   * of a key can not be deleted, only all versions of a given key at once.
+   */
   deleteKey(name: string, options?: coreHttp.OperationOptions): Promise<DeletedKey>;
+  /**
+   * The getDeletedKey method returns the specified deleted key along with its properties.
+   * This operation requires the keys/get permission.
+   */
   getDeletedKey(name: string, options?: GetDeletedKeyOptions): Promise<DeletedKey>;
 }
 
+/**
+ * As of http://tools.ietf.org/html/draft-ietf-jose-json-web-key-18
+ */
+export interface JsonWebKey {
+  /**
+   * Key identifier.
+   */
+  kid?: string;
+  /**
+   * JsonWebKey Key Type (kty), as defined in
+   * https://tools.ietf.org/html/draft-ietf-jose-json-web-algorithms-40. Possible values include:
+   * 'EC', 'EC-HSM', 'RSA', 'RSA-HSM', 'oct'
+   */
+  kty?: JsonWebKeyType;
+  /**
+   * @member {JsonWebKeyOperation[]} [keyOps] Json web key operations. For more
+   * information on possible key operations, see JsonWebKeyOperation.
+   */
+  keyOps?: JsonWebKeyOperation[];
+  /**
+   * RSA modulus.
+   */
+  n?: Uint8Array;
+  /**
+   * RSA public exponent.
+   */
+  e?: Uint8Array;
+  /**
+   * RSA private exponent, or the D component of an EC private key.
+   */
+  d?: Uint8Array;
+  /**
+   * RSA private key parameter.
+   */
+  dp?: Uint8Array;
+  /**
+   * RSA private key parameter.
+   */
+  dq?: Uint8Array;
+  /**
+   * RSA private key parameter.
+   */
+  qi?: Uint8Array;
+  /**
+   * RSA secret prime.
+   */
+  p?: Uint8Array;
+  /**
+   * RSA secret prime, with p < q.
```

Comment:
In .NET we removed the ", with p < q" such that `p` and `q` have the same comment.